tests/Benchmarks/Microsoft.Experimental.Collections/RefDictionaryPerformanceTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.Collections.Extensions.Tests
+{
+    [MemoryDiagnoser]
+    public class RefDictionaryPerformanceTests
+    {
+        [Params(100000)]
+        public int Size { get; set; }
+
+        ulong[] _keys;
+        int[] _values;
+
+        RefDictionary<ulong, int> _refDict;
+        Dictionary<ulong, int> _dict;
+
+        [GlobalSetup(Targets = new[] { nameof(LoadRefDictionary), nameof(LoadDictionary) })]
```

Comment:
I think if you don't specify targets in a GlobalSetup it will execute for all the benchmarks, but I don't know if you plan on adding more benchmarks.

Reply:
Yes, more work to do here. Its size sensitive for some reason, and I need to make sure the tests represents the benchmarksgame. It shows about 40% perf improvement on this size atm.